### ISSUE TITLE
Flarum is unable to Load if there are large number of tags

### ISSUE THREAD

**Issue by sanjibnarzary** [flarum/core#1264]
> Issues on Github are meant for bug reporting. Please post feature requests on the [discussion forum](https://discuss.flarum.org/t/features).
---
> Try to complete the below form as far as you are able and are willing to share. Add a screenshot of the issue if you can.

## Explanation
I have inserted 50000 tags which I have imported from stackexchange data dump in to the flarum tags table by using a script. My flarum site was working fine with 1000 number of tags but when I increase the number of tags it fail to load.  When I remove the imported tags it again works as normal. 

## Technical details

- Version of Flarum: 7.beta
- Website URL where the bug is visible: http://localhost
- The webserver you are running: apache
- PHP version: 7.0
- Hosted environment: local machine 

## Flarum info

```
Output of "php flarum info", run this in terminal in your Flarum directory.
```

## Log files

```
Put any relevant logs here.
```

**Comment by franzliedke** [comment]
Is that a legitimate use-case? What exactly do you want to achieve with such a large number of tags?

**Comment by tobscure** [comment]
I think we should support this use-case (eg. an extension which allows users to specify their own tags) but it's low priority.

I assume the bottleneck currently is that we load/display all tags at once by default.

**Comment by sanjibnarzary** [comment]
@franzliedke What if I wanted to create a website similar to stackexchange with flarum.
@tobscure Where do I change the code for displaying only a few tags?

**Comment by franzliedke** [comment]
Well, with such an extension, you probably wouldn't want to display all tags like we do (by default). I am just saying that this kind of use-case might fall outside of our tags extension's scope.

**Comment by tobscure** [comment]
Relevant discussion in https://github.com/flarum/core/issues/395

**Comment by tobyzerner** [comment]
Closing this to be addressed as part of flarum/issue-archive#393.
